lib/Modules/ADCharmUtils/ADCharmUtils.psm1

```diff
@@ -173,3 +184,36 @@ function Start-JoinDomain {
     Write-JujuInfo "ad-join returned EMPTY"
     return $false
 }
+
+function Start-ADInfoRelationJoinedHook {
+    $requiredCtxt = @{
+        "address" = $null;
+        "domainName" = $null;
+        "netbiosname" = $null;
+        "adcredentials" = $null;
+        "ad-group-sid" = $null
```

Comment:
Please remove this from AD. You can get this from any domain joined computer using:

```powershell
(Get-CimInstance win32_group -Filter "Name='GroupNameHere'").SID
```